src/main/java/org/jenkinsci/plugins/durabletask/FileMonitoringTask.java

```diff
@@ -114,6 +114,11 @@ protected FileMonitoringController launchWithCookie(FilePath workspace, Launcher
         charset = SYSTEM_DEFAULT_CHARSET;
     }
 
+    @CheckForNull public final String getCharset()
```

Comment:
No need to expose this in the API.
```suggestion
    @CheckForNull final String getCharset()
```
(or just remove `private` from the field)